src/solver/variable/include/antares/solver/variable/economy/lolp_base.h

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright 2007-2025, RTE (https://www.rte-france.com)
+ * See AUTHORS.txt
+ * SPDX-License-Identifier: MPL-2.0
+ * This file is part of Antares-Simulator,
+ * Adequacy and Performance assessment for interconnected energy networks.
+ *
+ * Antares_Simulator is free software: you can redistribute it and/or modify
+ * it under the terms of the Mozilla Public Licence 2.0 as published by
+ * the Mozilla Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Antares_Simulator is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Mozilla Public Licence 2.0 for more details.
+ *
+ * You should have received a copy of the Mozilla Public Licence 2.0
+ * along with Antares_Simulator. If not, see <https://opensource.org/license/mpl-2-0/>.
+ */
+
+#pragma once
+
+#include <antares/memory/memory.h>
+#include <antares/solver/variable/categories.h>
+#include <antares/solver/variable/container.h>
+#include <antares/solver/variable/state.h>
+#include <antares/solver/variable/storage/intermediate.h>
+#include <antares/solver/variable/storage/results.h>
+#include <antares/solver/variable/surveyresults.h>
+#include <antares/solver/variable/variable.h>
+#include <antares/study/area/area.h>
+#include <antares/study/area/links.h>
+#include <antares/study/study.h>
+
+namespace Antares::Solver::Variable::Economy
+{
+
+template<class Traits>
+struct VCardLOLP_Base
+{
+    //! Caption
+    static std::string Caption()
+    {
+        return Traits::Caption();
+    }
+
+    //! Unit
+    static std::string Unit()
+    {
+        return Traits::Unit();
+    }
```

Comment:
Identical for the 2 traits
```suggestion
    static std::string Unit()
    {
        return "%";
    }
```

Reply:
`Traits::Unit()` was defined for each trait but no longer called, so I removed them.

https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/3144/commits/228060e6f0e2b4294f4e76cc2cd741c91c2ea563